src/AdminConsole/Endpoints/ApplicationEndpoints.cs

```diff
@@ -13,8 +13,18 @@ public static IEndpointRouteBuilder MapApplicationEndpoints(this IEndpointRouteB
         // The Blazor SSR sample uses the same solution for signing out, but we do not want to use all the endpoints.
         group.MapGet("/{appId}/Available", async ([AsParameters] GetAppIdAvailabilityRequest request, IPasswordlessManagementClient client) =>
         {
-            var result = await client.IsApplicationIdAvailableAsync(request);
-            return Ok(result);
+            try
+            {
+                var result = await client.IsApplicationIdAvailableAsync(request);
+                return Ok(result);
+            }
+            catch (PasswordlessApiException e)
+            {
+                return Json(
+                    e.Details,
+                    contentType: "application/problem+json",
+                    statusCode: e.Details.Status);
+            }
```

Comment:
In API, we do this inside a middleware for all endpoints. Maybe we should do that here too?

Reply:
@Tyrrrz Not sure I'm thinking correctly, but: Adminconsole doesn't return Json in most cases (this being a bit special endpoint). I think(?) we have a generic exception page that gets rendered for uncaught exceptions

Reply:
Ah I see. Makes sense.